.eslintrc

```
@@ -0,0 +1,25 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "rules": {
+        "no-console": 2, // Remember, this means error!
```

Explicit `error` works for me. I wonder are you using the integer here?

```suggestion
        "no-console": "error",
```

---

Thanks for the feedback, got updated on various things :+1: 

I copy/pasted (:scream:)  from another project to get something working quickly